etc/config/hlsl.amazon.properties

```diff
@@ -15,3 +11,37 @@ compiler.dxc_1_6_2112.exe=/opt/compiler-explorer/dxc-1.6.2112/bin/dxc
 compiler.dxc_1_6_2112.semver=1.6.2112
 compiler.dxc_1_7_2207.exe=/opt/compiler-explorer/dxc-1.7.2207/bin/dxc
 compiler.dxc_1_7_2207.semver=1.7.2207
+
+group.rga.compilers=rga262_dxctrunk:rga262_dxc172207:rga262_dxc162112:rga261_dxc172207:rga261_dxc162112
+group.rga.groupName=RGA
+group.rga.isSemVer=true
+group.rga.compilerType=rga
+
+compiler.rga262_dxctrunk.exe=/opt/compiler-explorer/rga-2.6.2/rga
```

Comment:
These paths will need updating. The installation unpacked them:

```
Aug/17 12:33 admin-node~/infra (main|✔) 4.8s $ ls -ld /opt/compiler-explorer/rga-*
drwxrwxr-x 6 1001 1002 6144 May 18 14:12 /opt/compiler-explorer/rga-2.6.1.23/
drwxrwxr-x 7 1001 1002 6144 Jul 25 14:56 /opt/compiler-explorer/rga-2.6.2.38/
```
But the compiler is in a direct subdirectory:
```
Aug/17 12:33 admin-node~/infra (main|✔) $ /opt/compiler-explorer/rga-2.6.1.23/rga --version
/opt/compiler-explorer/rga-2.6.1.23/rga

Radeon GPU Analyzer Version: 2.6.1.23
```